requirements.txt

```diff
@@ -1,3 +1,4 @@
 weaviate-client>=2.1.0
```

Comment:
Since we dropped the support for `weaviate-client` version 2.x.x. There were some bugs in the python-client so lets use the `3.7.0` version:
```python
        "weaviate-client>=3.7.0",
```